gateway/api/openapi/types.go

```diff
@@ -667,6 +667,10 @@ type ProxyManagerResponse struct {
 	Status ClientStatusType `json:"status"`
 	// The requested connection name
 	RequestConnectionName string `json:"connection_name"`
+	// The requested connection type
+	RequestConnectionType string `json:"connection_type"`
+	// The requested connection subtype
+	RequestConnectionSubType string `json:"connection_subtype"`
```

Comment:
You need to mark them as read only with the tag: `readonly:"true"`.
Make sure also to generate the docs when these files are changed:

```sh
make generate-openapi-docs
```

Reply:
done